CoEpi/ui/alerts/AlertCell.swift

```diff
@@ -1,3 +1,40 @@
-class AlertCell {
+import UIKit
 
+class AlertCell: UITableViewCell {
+    private static let nibName = "AlertView"
+
+    public var onAcknowledged: ((Alert) ->())?
+
+    private var alert: Alert?
+    private var alertView: AlertView?
+
+    public func setAlert(alert: Alert) {
+        self.alert = alert
+        setupUI(alert: alert)
+    }
+    
+    private func setupUI(alert: Alert) {
+        let view: AlertView = AlertView.fromNib()
+        view.onAcknowledged = { [weak self] in
+            guard let this = self, let alert = this.alert else { return }
```

Comment:
You don't need these, just
```swift
view.onAcknowledged = { [weak self] in
    self?.onAcknowledged?(alert)
}
```
Also, let's use `let self = self`, for consistency